libbindgen/src/codegen/mod.rs

```diff
@@ -274,22 +269,28 @@ impl CodeGenerator for Module {
                item: &Item) {
         debug!("<Module as CodeGenerator>::codegen: item = {:?}", item);
 
-        if !ctx.options().enable_cxx_namespaces {
-            for child in self.children() {
-                ctx.resolve_item(*child)
-                    .codegen(ctx, result, whitelisted_items, &());
+        let codegen_self = |result: &mut CodegenResult| {
+            // FIXME: This could be less expensive, I guess.
```

Comment:
Gah, I think you're totally right, I just added this because initially I planed to generate also nested items, but that didn't work before and shouldn't work after reflection.